### PR TITLE
Migrate to validated variable names

### PR DESCRIPTION
**Pull request by bzg**
This is still WIP to get the schema completely aligned with the validated one.

### CODE REVIEW

**Comment by bonjourmauko** [comment]
I update in parallel tests, algorithm and generated spec.

File: lib/covid19_orientation_web/schemas/conclusion.ex [hunk 1]
```diff
@@ -14,14 +14,14 @@ defmodule Covid19OrientationWeb.Schemas.Conclusion do
     properties: %{
       code: %Schema{
         type: :string,
-        enum: ["FIN1", "FIN2", "FIN3", "FIN4", "FIN5", "FIN6", "FIN7", "FIN8", "FIN9"],
+        enum: ["orientation_SAMU", "FIN2", "FIN3", "FIN4", "FIN5", "FIN6", "FIN7", "FIN8", "FIN9"],
```
Comment (bonjourmauko) [inline lib/covid19_orientation_web/schemas/conclusion.ex]: Do you have the reference ?

Comment (bzg) [reply lib/covid19_orientation_web/schemas/conclusion.ex]: Column B72 of the reference spreadsheet: "orientation" => "FIN1". This is the only orientation taken into account in the spreadsheet, I read this as "other orientations don't need to be stored", which makes sense, as long as all data are collected. WDYT?

Comment (bonjourmauko) [reply lib/covid19_orientation_web/schemas/conclusion.ex]: I think you got C73? I'd rather store them, also is simpler.

Comment (bonjourmauko) [reply lib/covid19_orientation_web/schemas/conclusion.ex]: « Exceptions are always the source of more exceptions »

Comment (bzg) [reply lib/covid19_orientation_web/schemas/conclusion.ex]: Mhh.. no, I got B72, see below.

B72

Comment (bzg) [reply lib/covid19_orientation_web/schemas/conclusion.ex]: Agreed about the exceptions... except that in this case, the `orientation_SAMU` variable name seems (deadly) critical!

**Pull request merged**